.github/workflows/code_scan.yml

```diff
@@ -9,38 +9,26 @@ on:
 jobs:
   Trivy-scan:
     runs-on: ubuntu-20.04
-    permissions:
-      security-events: write
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Run Trivy vulnerability scanner in repo mode
-        uses: aquasecurity/trivy-action@0.9.2
-        with:
-          scan-type: fs
-          ignore-unfixed: true
-          format: sarif
-          output: trivy-results.sarif
-          severity: CRITICAL
-      - name: Upload Trivy scan results to GitHub Security tab
-        uses: github/codeql-action/upload-sarif@v2
+      - name: Set up Python
+        uses: actions/setup-python@v4
         with:
-          sarif_file: trivy-results.sarif
-  Trivy-gen-sbom:
-    runs-on: ubuntu-20.04
-    permissions:
-      contents: write
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-      - name: Run Trivy in GitHub SBOM mode and submit results to Dependency Graph
-        uses: aquasecurity/trivy-action@0.9.2
+          python-version: "3.10"
+      - name: Install dependencies
+        run: python -m pip install tox
```

Comment:
I'm not sure but isn't it necessary to consider tox version? because I watched tox is installed as below in other part.
```
python -m pip install -r requirements/dev.txt
```

Reply:
@eunwoosh In case of testing, need to install testing dependency as well using the requirements/dev.txt but for the code scanning, we don't need to install them. only required thing is all list of dependencies. So, I think it could reduce the time to be done for this workflow.